pyupgrade/_plugins/legacy.py

```diff
@@ -28,14 +28,6 @@
 FUNC_TYPES = (ast.Lambda, ast.FunctionDef, ast.AsyncFunctionDef)
 
 
-def _fix_yield(i: int, tokens: List[Token]) -> None:
```

Comment:
Gotcha, so `yield_from` and `_to_fstring` are completely gone as you said in description.